docs/dev/app/misc.md

```diff
@@ -0,0 +1,20 @@
+# Miscellaneous 
```

Comment:
What if we renamed this page/file to "Styling"/"styling.md"? I feel like adding or changing icons could fall under styling, and we could also include info about global styles, text/font in here (maybe we can add these on a diff branch). Miscellaneous may be a bit too vague.

Reply:
gotcha, i'll change the page to styling.md as suggested for now, and we can flesh it out on the next PR?